apps/studio/.storybook/preview.tsx

```diff
@@ -140,41 +141,8 @@ const LoginStateDecorator: Decorator<Args> = (story, { parameters }) => {
   )
 }
 
-export const MockDateDecorator: Decorator<Args> = (story, { parameters }) => {
-  mockdate.reset()
-
-  if (!parameters.mockdate) {
-    return story()
-  }
-
-  // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
-  mockdate.set(parameters.mockdate)
```

Comment:
question: hmm I'm not sure if we will need it but this change seems to remove the ability to set a custom date on individual stories? From the docs seems like they say to put the date parameter on the stories rather than the global one here

Or am I reading it wrong haha please correct me if I interpret this incorrectly

Reply:
Ok nice, thanks!